components/director/internal/tenantmapping/cert_service_context_provider.go

```diff
@@ -91,3 +126,15 @@ func (m *certServiceContextProvider) Match(_ context.Context, data oathkeeper.Re
 
 	return false, nil, nil
 }
+
+func (p *certServiceContextProvider) directorScopes(consumerType model.SystemAuthReferenceObjectType) (string, error) {
+	if consumerType == "" {
+		consumerType = "default"
```

Comment:
What is the use-case for the default after this change?

Reply:
BAS flow :alien: - those scopes were hardcoded before as default